elmui/src/Main.elm

```diff
@@ -0,0 +1,62 @@
+port module Main exposing (..)
+
+import Browser
+import Html exposing (Html, Attribute, button, div, input, text, textarea)
+import Html.Attributes exposing (..)
```

Comment:
You could tighten up `..`

Reply:
I'm going to leave it for now since this is such a skeleton. I'll hack on it more. I'm thinking of trying: http://elm-bootstrap.info/ to make it fancier. Have you tried that library?

What do you recommend for better looking UIs?

Reply:
That's what I've used. In particular Bootstrap.Grid and Bootstrap.Tab were helpful.